src/stp/Play.cpp

```diff
@@ -117,24 +118,23 @@ std::unordered_map<Role*, Status> const& Play::getRoleStatuses() const { return
 
 bool Play::isValidPlayToKeep(PlayEvaluator& playEvaluator) noexcept {
     if (!interface::MainControlsWidget::ignoreInvariants) {
-        return std::all_of(keepPlayEvaluation.begin(), keepPlayEvaluation.end(), [playEvaluator] (auto& x) { return playEvaluator.checkEvaluation(x); });
+        return std::all_of(keepPlayEvaluation.begin(), keepPlayEvaluation.end(), [&playEvaluator] (auto& x) { return playEvaluator.checkEvaluation(x); });
```

Comment:
Consider formatting this:

```
return std::all_of(begin(), end(), [data](arg) {
    return ...;
});
```

Reply:
I only see a change in an enter in the lambda, is that correct?

Reply:
Correct. it's kinda gross on 1 line.